src/commands/sourcemaps/upload.ts

```diff
@@ -122,6 +123,17 @@ export class UploadCommand extends Command {
   }
 
   private getMinifiedURL(minifiedFilePath: string): string {
+    let protocol
+    try {
+      const objUrl = new URL(this.minifiedPathPrefix!)
+      protocol = objUrl.protocol
+    } catch {
+      // Do nothing.
+    }
+
+    if (!protocol && !this.minifiedPathPrefix!.startsWith('/')) {
+      throw new Error('Absolute path must have a leading slash')
```

Comment:
With this version if we use the command line incorrectly we only have this error message + a stack trace, which is not very helpful:
```
Internal Error: Absolute path must have a leading slash
    at UploadCommand.getMinifiedURL (/Users/loic.raucy/go/src/github.com/DataDog/datadog-ci/src/commands/sourcemaps/upload.ts:135:13)
    at /Users/loic.raucy/go/src/github.com/DataDog/datadog-ci/src/commands/sourcemaps/upload.ts:116:27
    at Array.map (<anonymous>)
    at UploadCommand.getMatchingSourcemapFiles (/Users/loic.raucy/go/src/github.com/DataDog/datadog-ci/src/commands/sourcemaps/upload.ts:111:27)
    at UploadCommand.<anonymous> (/Users/loic.raucy/go/src/github.com/DataDog/datadog-ci/src/commands/sourcemaps/upload.ts:89:27)
    at Generator.next (<anonymous>)
    at /Users/loic.raucy/go/src/github.com/DataDog/datadog-ci/src/commands/sourcemaps/upload.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/loic.raucy/go/src/github.com/DataDog/datadog-ci/src/commands/sourcemaps/upload.ts:4:12)
    at UploadCommand.execute (/Users/loic.raucy/go/src/github.com/DataDog/datadog-ci/src/commands/sourcemaps/upload.ts:40:16)
error Command failed with exit code 1.
```

We could avoid rendering the stack trace, render a more complete error message (something that gives more context such as `--minified-path-prefix should either be an URL (such as "http://example.com/static") or an absolute path starting with a `/` such as "/static")` displayed in red and exit with a non-zero status code.

We already have some rendering code in `src/commands/sourcemaps/renderer.ts` and other commands might have more examples.

Reply:
Thanks for the `renderer.ts` tip. Changed the code a bit to accommodate the proper error message.